bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableTableAdminClientWrapper.java

```diff
@@ -0,0 +1,251 @@
+
+    doNothing().when(mockAdminClient).deleteTable(request);
+    clientWrapper.deleteTable(TABLE_ID);
+
+    verify(mockAdminClient).deleteTable(request);
+  }
+
+  @Test
+  public void testDeleteTableAsync(){
+    DeleteTableRequest request = DeleteTableRequest.newBuilder().setName(TABLE_NAME).build();
+
+    when(mockAdminClient.deleteTableAsync(request))
+        .thenReturn(immediateFuture(Empty.newBuilder().build()));
+    clientWrapper.deleteTableAsync(TABLE_ID);
+
+    verify(mockAdminClient).deleteTableAsync(request);
+  }
+
+  @Test
+  public void testModifyFamilies(){
+    ModifyColumnFamiliesRequest request =
+        ModifyColumnFamiliesRequest
+            .of(TABLE_ID)
+            .addFamily(COLUMN_FAMILY, GCRULES.maxVersions(1))
+            .updateFamily("update-family", GCRULES.defaultRule());
+
+    when(mockAdminClient.modifyColumnFamily(request.toProto(INSTANCE_NAME.toAdminInstanceName())))
+        .thenReturn(createTableData());
+    Table response = clientWrapper.modifyFamilies(request);
+
+    assertEquals(Table.fromProto(createTableData()), response);
+    verify(mockAdminClient).modifyColumnFamily(request.toProto(INSTANCE_NAME.toAdminInstanceName()));
+  }
+
+  @Test
+  public void testModifyFamiliesAsync() throws Exception{
+    ModifyColumnFamiliesRequest request =
+        ModifyColumnFamiliesRequest
+            .of(TABLE_ID)
+            .addFamily(COLUMN_FAMILY, GCRULES.maxVersions(1))
+            .updateFamily("update-family", GCRULES.defaultRule());
+
+    when(mockAdminClient.modifyColumnFamilyAsync(request.toProto(INSTANCE_NAME.toAdminInstanceName())))
+        .thenReturn(immediateFuture(createTableData()));
+    ListenableFuture<Table> response = clientWrapper.modifyFamiliesAsync(request);
+
+    assertEquals(Table.fromProto(createTableData()), response.get());
+    verify(mockAdminClient).modifyColumnFamilyAsync(request.toProto(INSTANCE_NAME.toAdminInstanceName()));
+  }
+
+  @Test
+  public void dropRowRange(){
+    DropRowRangeRequest request = DropRowRangeRequest.newBuilder()
+        .setName(TABLE_NAME)
+        .setRowKeyPrefix(ByteString.copyFromUtf8(ROW_KEY_PREFIX))
+        .build();
+
+    doNothing().when(mockAdminClient).dropRowRange(request);
+    clientWrapper.dropRowRange(TABLE_ID, ROW_KEY_PREFIX);
+
+    verify(mockAdminClient).dropRowRange(request);
+  }
+
+  @Test
+  public void dropRowRangeAsync(){
+    DropRowRangeRequest request = DropRowRangeRequest.newBuilder()
+        .setName(TABLE_NAME)
+        .setRowKeyPrefix(ByteString.copyFromUtf8(ROW_KEY_PREFIX))
+        .build();
+
+    when(mockAdminClient.dropRowRangeAsync(request))
+        .thenReturn(immediateFuture(Empty.newBuilder().build()));
+    clientWrapper.dropRowRangeAsync(TABLE_ID, ROW_KEY_PREFIX);
+
+    verify(mockAdminClient).dropRowRangeAsync(request);
+  }
+
+  private static com.google.bigtable.admin.v2.Table createTableData(){
```

Comment:
can you use import com.google.bigtable.admin.v2.Table instead ?

Reply:
This class uses `com.google.cloud.bigtable.admin.v2.models.Table` as well as `com.google.bigtable.admin.v2.Table`, So I could not import it.